### DISCUSSION TITLE
Creation of components

### DISCUSSION THREAD

**Discussion by AyphixLLC** [nxrighthere/UnrealCLR#222] [Help] [answered]
I'm attempting to port my ability framework to C#, so far I've been completely unable to derive from ActorComponent, so I searched in the discussions and found an answer saying to use SceneComponent, but this has the same issue. Is there any way to create a component that I can put on my characters similar to an ActorComponent? I'm using the newest version of UE and UnrealCLR.

**Comment by nxrighthere** [comment, marked as answer]
Could you please elaborate on how you expect it to work? What are you trying to do, and what result do you expect?

**Reply by AyphixLLC** [reply]
I would like to be able to create an ActorComponent from C# that I can apply to my character in-editor. Reading further into this I'm starting to realize that managed code can only be accessed at Runtime am I wrong in this assumption? So what I'm asking is probably impossible from the managed world.

**Reply by nxrighthere** [reply]
Yes, exactly. There's no representation of a managed class in the editor at the moment.
